icons/jar.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   version="1.1"
+   width="24"
+   height="24"
+   viewBox="0 0 24 24"
+   id="svg4"
+   sodipodi:docname="square-rounded.svg"
+   inkscape:version="1.1 (c68e22c387, 2021-05-23)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs8" />
+  <sodipodi:namedview
+     id="namedview6"
+     pagecolor="#505050"
+     bordercolor="#ffffff"
+     borderopacity="1"
+     inkscape:pageshadow="0"
+     inkscape:pageopacity="0"
+     inkscape:pagecheckerboard="1"
+     showgrid="false"
+     inkscape:object-paths="true"
+     inkscape:snap-intersection-paths="true"
+     inkscape:snap-object-midpoints="true"
+     inkscape:object-nodes="true"
+     inkscape:snap-others="true"
+     inkscape:zoom="22.627417"
+     inkscape:cx="0.64081552"
+     inkscape:cy="13.103573"
+     inkscape:window-width="1600"
+     inkscape:window-height="847"
+     inkscape:window-x="-8"
+     inkscape:window-y="-8"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg4" />
+  <path
+     id="path2"
+     style="fill:#f44336;fill-opacity:1;stroke-width:1.14099"
+     d="M 7.4355469 1.7304688 C 4.286423 1.7304687 1.7304688 4.286423 1.7304688 7.4355469 L 1.7304688 16.564453 C 1.7304687 19.713577 4.286423 22.269531 7.4355469 22.269531 L 16.564453 22.269531 C 19.713577 22.269531 22.269531 19.713577 22.269531 16.564453 L 22.269531 7.4355469 C 22.269531 4.286423 19.713577 1.7304688 16.564453 1.7304688 L 7.4355469 1.7304688 z M 6.0371094 6.2128906 L 16.548828 6.2128906 L 18.203125 6.2128906 C 19.112626 6.2128906 19.855469 6.9168842 19.855469 7.7929688 L 19.855469 10.160156 A 1.6536387 1.5785279 0 0 1 18.203125 11.736328 L 16.548828 11.736328 L 16.548828 12.470703 L 16.548828 13.453125 A 3.0036731 2.896588 0 0 1 13.544922 16.351562 L 9.0410156 16.351562 A 3.0036731 2.896588 0 0 1 6.0371094 13.453125 L 6.0371094 6.2128906 z M 16.548828 7.7929688 L 16.548828 9.8339844 L 16.548828 10.160156 L 18.203125 10.160156 L 18.203125 7.7929688 L 16.548828 7.7929688 z M 4.2753906 17.763672 L 19.503906 17.763672 L 19.503906 19.417969 L 4.2753906 19.417969 L 4.2753906 17.763672 z " />
```

Comment:
This seems overly complex. A jar is an svg so we can take inspiration from `zip.svg`:

```suggestion
     d="M 5 3 C 3.9 3 3 3.9 3 5 L 3 19 C 3 20.1 3.9 21 5 21 L 19 21 C 20.1 21 21 20.1 21 19 L 21 5 C 21 3.9 20.1 3 19 3 L 5 3 z M 6 7 L 16 7 C 17 7 18 8 18 9 L 18 11 C 18 12 17 13 16 13 L 14 13 L 14 15 C 14 16 13 17 12 17 L 8 17 C 7 17 6 16 6 15 L 6 7 z M 14 9 L 14 11 L 16 11 L 16 9 L 14 9 Z" />
```